bindings/python/multibody/fcl/expose-fcl.cpp

```diff
@@ -18,18 +18,84 @@ namespace pinocchio
     void exposeFCL()
     {
       using namespace pinocchio::python::fcl;
-      ContactPythonVisitor::expose();
-      StdVectorPythonVisitor<ContactPythonVisitor::Contact>::expose("StdVec_Contact");
+      namespace bp = boost::python;
       
-      CollisionResultPythonVisitor::expose();
-      StdVectorPythonVisitor<CollisionResultPythonVisitor::CollisionResult>::expose("StdVec_CollisionResult");
+      if(!eigenpy::register_symbolic_link_to_registered_type< ::hpp::fcl::OBJECT_TYPE>())
+      {
+        bp::enum_< ::hpp::fcl::OBJECT_TYPE >("OBJECT_TYPE")
+        .value("OT_UNKNOWN",::hpp::fcl::OT_UNKNOWN)
+        .value("OT_BVH",::hpp::fcl::OT_BVH)
+        .value("OT_GEOM",::hpp::fcl::OT_GEOM)
+        .value("OT_OCTREE",::hpp::fcl::OT_OCTREE)
+        .value("OT_COUNT",::hpp::fcl::OT_COUNT)
+        ;
+      }
+        
+      if(!eigenpy::register_symbolic_link_to_registered_type< ::hpp::fcl::NODE_TYPE>())
+      {
+        bp::enum_< ::hpp::fcl::NODE_TYPE >("NODE_TYPE")
+        .value("BV_UNKNOWN",::hpp::fcl::BV_UNKNOWN)
+        .value("BV_AABB",::hpp::fcl::BV_AABB)
+        .value("BV_OBB",::hpp::fcl::BV_OBB)
+        .value("BV_RSS",::hpp::fcl::BV_RSS)
+        .value("BV_kIOS",::hpp::fcl::BV_kIOS)
+        .value("BV_OBBRSS",::hpp::fcl::BV_OBBRSS)
+        .value("BV_KDOP16",::hpp::fcl::BV_KDOP16)
+        .value("BV_KDOP18",::hpp::fcl::BV_KDOP18)
+        .value("BV_KDOP24",::hpp::fcl::BV_KDOP24)
+        .value("GEOM_BOX",::hpp::fcl::GEOM_BOX)
+        .value("GEOM_SPHERE",::hpp::fcl::GEOM_SPHERE)
+        .value("GEOM_CAPSULE",::hpp::fcl::GEOM_CAPSULE)
+        .value("GEOM_CONE",::hpp::fcl::GEOM_CONE)
+        .value("GEOM_CYLINDER",::hpp::fcl::GEOM_CYLINDER)
+        .value("GEOM_CONVEX",::hpp::fcl::GEOM_CONVEX)
+        .value("GEOM_PLANE",::hpp::fcl::GEOM_PLANE)
+        .value("GEOM_HALFSPACE",::hpp::fcl::GEOM_HALFSPACE)
+        .value("GEOM_TRIANGLE",::hpp::fcl::GEOM_TRIANGLE)
+        .value("GEOM_OCTREE",::hpp::fcl::GEOM_OCTREE)
+        .value("NODE_COUNT",::hpp::fcl::NODE_COUNT)
+        ;
```

Comment:
Do you need to re-expose them ?

Reply:
I may not understand what `eigenpy::register_symbolic_link_to_registered_type` does.

Reply:
When you have implemented the Python bindings inside hpp-fcl, I don't know how much you realized that some part was already in Pinocchio.
So, the idea is to keep these bindings inside Pinocchio for a certain fixed duration (and to remove them in 6 months for instance). To avoid the duplication of symbols, I have implemented in EigenPy register_symbolic_link_to_registered_type<Type> which:
- check if the Type has been registered
- if it has been registered, then just put a symlink in the current Python scope
- otherwise, return False.
In the present context, if the result is False, it will just expose the enum as it was done before.
Is it OK for you?

Reply:
It's ok. Thanks.